demo-app/src/component.js

```diff
@@ -13,7 +13,9 @@ window.onhashchange = function() {
 // Start intercepting link click events for routing
 let iframeClient = new Client();
 iframeClient.start();
-
```

Comment:
Normally I'd say the promise API is the way to go, but we'd have to polyfill for IE11 and I don't want this library to polyfill normally built-in in browser features. I think setting a callback for this specific use case is fine. It should be a one time event, unlike pub-sub type data which can happen many times, so the contract is a little different.

Reply:
Sounds good to me.  On a related note I think if thats our goal, we may have to look at what our js output is.  Since I believe we are still using Set's which would require a polyfill even with Babel (IIRC).

Reply:
Good call on the use of `Set`. I created a story to follow up on that in general.

Reply:
> Normally I'd say the promise API is the way to go, but we'd have to polyfill for IE11 and I don't want this library to polyfill normally built-in in browser features. I think setting a callback for this specific use case is fine. It should be a one time event, unlike pub-sub type data which can happen many times, so the contract is a little different.

:point_up: I'm reconsidering my thoughts on emitting this as an event after looking at the pub-sub API proposal. See my comments there for more.